### DISCUSSION TITLE
Lİcencing question

### DISCUSSION THREAD

**Discussion by kaansenol** [OFFLINE-GmbH/oc-boxes-support#146] [Help] [answered]
Hi,
We are building a project for a client using your plugin. We have two instances one is test and the other is prod. Both are under different domains. When we try to merge the test with prod we are having licensing issues. How can we solve this problem? Is there any possibility of using the same license for both instances?

**Comment by tobias-kuendig** [comment, marked as answer]
@kaansenol you license the plugin per project you register with October here: https://octobercms.com/account/project/dashboard. This means it does not matter on how many domains you install the plugin, as long as they belong to the same October project.

In short, you should not run into any issues when using the plugin on two domains. It might be that you have another problem with your setup.

Maybe share some more error details so we can narrow down the issue.

**Comment by kaansenol** [comment]
Thanks for the explanation...
